src/containers/settings/EditServiceScreen.tsx

```diff
@@ -151,10 +149,10 @@ class EditServiceScreen extends Component<EditServicesScreenProps> {
   prepareForm(recipe: IRecipe, service: Service | null, proxy: any): Form {
     const { intl } = this.props;
 
-    const { stores, router } = this.props;
+    const { stores } = this.props;
 
     // @ts-ignore TODO: This is actually there and we don't have a correct type right now.
```

Comment:
Is this ignore still needed?

Reply:
Will make sure to work this out as well in this PR later.

Reply:
This has been resolved, ts-ignores are deleted and the line below updated to correctly use the useParams hook (added a wrapper component to be able to use that hook within a class environment)